openedx/features/caliper_tracking/tests/current/edx.forum.comment.created.json

```diff
@@ -0,0 +1,41 @@
+{
+  "accept_language": "en-US,en;q=0.5",
+  "agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:62.0) Gecko/20100101 Firefox/62.0",
+  "context": {
+    "course_id": "course-v1:edX+DemoX+Demo_Course",
+    "course_user_tags": {},
+    "org_id": "edX",
+    "path": "/courses/course-v1:edX+DemoX+Demo_Course/discussion/comments/5bc889c5977195008e000003/reply",
+    "user_id": 13
+  },
+  "event": {
+    "body": "This is another comment",
+    "commentable_id": "i4x-edx-eiorguegnru-course-foobarbaz",
+    "discussion": {
+      "id": "5bc887e1977195008e000000"
+    },
+    "id": "5bc88be5977195008e000004",
+    "options": {
+      "followed": false
+    },
+    "response": {
+      "id": "5bc889c5977195008e000003"
+    },
+    "truncated": false,
+    "url": "http://edx.devstack.lms:18000/courses/course-v1:edX+DemoX+Demo_Course/discussion/forum/i4x-edx-eiorguegnru-course-foobarbaz/threads/5bc887e1977195008e000000",
```

Comment:
Please replace all occurrences of `edx.devstack.lms` with `localhost`.

Reply:
done